service/synapse-service-graphql/pom.xml

```diff
@@ -38,17 +38,21 @@
         </dependency>
 
         <!--GraphQL-->
+        <dependency>
+            <groupId>com.graphql-java</groupId>
+            <artifactId>graphql-java-extended-scalars</artifactId>
+        </dependency>
```

Comment:
Is there a reason why this is added?

Reply:
This is needed for graphql to work, reverting the changes that was done in previous pr when upgrading to springboot3